app/scripts/requirements

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+function colorizeOutput($text, $color)
+    {
+        switch ($color) {
+            case 'green':
+                // White on green
+                $output = "\033[1;37m\033[42m{$text}";
+                break;
+            case 'red':
+                // White on red
+                $output = "\033[1;37m\033[41m{$text}";
+                break;
+            default:
+                $output = $text;
+                break;
+        }
+        $output .= "\033[0m";
+        return $output;
+    }
+
+/**
+ * Checks a configuration.
+ */
+function check($boolean, $message, $help = '', $fatal = false, $fix = '')
+{
+    $text = sprintf("* $message%s\n", $boolean ? '' : ': FAILED');
+
+    if (! $boolean) {
+        if(! empty($help)) { 
+            $text .=  "  $help\n";
+        }
+        if(! empty($fix)) {
+                $text .=  "  $fix\n";
+        }
+    }
+    if($boolean) {
+        echo colorizeOutput($text, 'green');
```

Comment:
Too much text with colored background. Let's color only the result, e.g. OK, FAILED, and keep the rest standard.